packages/shorebird_cli/lib/src/logger.dart

```diff
@@ -42,6 +42,11 @@ void _writeToLogFile(Object? message, {required File logFile}) {
     return;
   }
 
+  // Making sure the log file exists before writing to it.
+  // This is necessary because the log file may be deleted by a cache
+  // clear command.
+  if (!logFile.existsSync()) return;
```

Comment:
My reasoning on this: The file should always be there, since it is created upon initialization. So if it is not there, that means that we shouldn't log, i.g. on the cache clear commando I don't think we should recreate the file that we just created.

Lmk if you all think that this makes sense or if you can think in a better approach.

Reply:
I don't know that I'm convinced that `cache clear` should delete logs. Maybe that's a cleaner solution? I worry this might put us in a silent failure scenario.

Reply:
humm, yeah, that is a good argument! logs are not cache! I am gonna instead remove the log clearing call from the command then!

Reply:
Updated, lmk wdyt!